src/vg_cluster_mgr.erl

```diff
@@ -118,3 +129,38 @@ head([{_, Host, _, ClientPort} | _]) ->
 
 tail(Nodes) ->
     head(lists:reverse(Nodes)).
+
+ensure_all_topics(ChainName, State) ->
+    Topics = vg_utils:topics_on_disk(),
```

Comment:
This isn't actually going to work when there are more than one chains, is it?  It assumes that the local node has all the correct topics on disk, but since there will only be one manager, that's only true for one chain, right?